.gitignore

```diff
@@ -12,6 +12,8 @@ trunk/NDHMS/HRLDAS/user_build_options
 /.project
 trunk/NDHMS/Run
 trunk/NDHMS/Run/
+trunk/NDHMS/NWM/
```

Comment:
This looks custom. Unless I'm mistaken. It could be removed and placed in an untracked .gitignore inside the/your untracked trunk/NDHMS/NWM repository

Reply:
Okay thanks. I will change this.

Reply:
Not a biggie, but every line matters and i'm trying to build peoples habits of carefully reviewing their git diffs for PRs. Doing this, building this practice should help result in "bite sized" PRs.

Reply:
I would also say that historically, and I would say continuing, we have a bad habbit of letting in too much unnecessary code. This is probably even more about the reviewers than the PR.

Reply:
Good points. I will do that from now on.